project_issue_task/models/project_issue.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 - 2013 Daniel Reis
+# Copyright 2016 - Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models, _
+
+
+class ProjectIssue(models.Model):
+    _inherit = 'project.issue'
+
+    @api.multi
+    def action_create_task(self):
+        """
+        Create and a related Task for the visit report, and open it's Form.
+        """
+        res = {}
+        for rec in self:
+
+            assert not rec.task_id, _("A Task is already assigned to the "
+                                      "Issue!")
+
+            rec_fields = ['project_id', 'analytic_account_id', 'location_id']
+            task_data = dict([(x, getattr(rec, x).id) for x in rec_fields
+                              if hasattr(rec, x) and getattr(rec, x)])
+            task_data['name'] = _('Report for %s') % rec.name
+            task_data['issue_id'] = rec.id
+            task_data['tag_ids'] = [(6, 0, [x.id for x in rec.tag_ids])]
+
+            task_model = self.env['project.task']
+            task_id = task_model.create(task_data)
+            rec.write({'task_id': task_id.id})
+            res = {
```

Comment:
This only opens the last created task. Two possibilities:

* Put `self.ensure_one()` and prevent multi-records.
* Add the task to a list, and at the end open a tree or a form view depending on the number of records.